integration_test/third_party_apps_data/test_config.yaml

```diff
@@ -41,3 +41,5 @@ per_application_overrides:
     platforms_to_skip: *common_skips
   redis:
     platforms_to_skip: *common_skips
+  memcached:
+    platforms_to_skip: *common_skips
```

Comment:
For newly added applications, we should not skip these distros. The older ones were released before we have cross distro tests. At some point, we'd need to go fix known issues in them as well.

Reply:
Looks like this has been addressed in https://github.com/GoogleCloudPlatform/ops-agent/pull/278/commits/99511cc178725c45d81ff56865a3c563ea8040a7

Reply:
yes, but haven't been able to verify it works.